openpdf/src/main/java/com/lowagie/text/pdf/PdfAppearance.java

```diff
@@ -55,8 +57,8 @@
  */
 
 public class PdfAppearance extends PdfTemplate {
-    
-    public static final HashMap stdFieldFontNames = new HashMap();
```

Comment:
Removing public access might (we don't know) break someones setup. Meaning if we include such changes it should by Semantic versioning be a Major version change. But, when something that should be internal was `public` by accident is fixed I'm so not sure.

Reply:
Oh, I Understand. I'll revert th "public" removals. That is a good point.

Reply:
We can take all of these issues that are breaking changes in the same release as the package renaming needed for Maven Central.

Reply:
I think the change was really wrong. If the method was public, then it's part of the API, should be documented and there should also be a test for that, so the method will be used. But there is a thing about writing tests for legacy code :-( . Anyway, I'd like to generify the code more.